.changelog/unreleased/improvement/279-remove-legacy-api.md

```diff
@@ -0,0 +1,2 @@
+- Remove Reader and Keeper API ([#279](https://github.com/cosmos/ibc-
```

Comment:
Isn't this a breaking change?

Reply:
definitively [5904117](https://github.com/cosmos/ibc-rs/pull/456/commits/590411795a5b82fe099050b6c9ae19fe058ed209)